### PR TITLE
UI bux fixess like hover effects, styles, text

### PR DESCRIPTION
**Pull request by DevSaheerHost**
This is more healpfull for ui

`<>used<>`

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -76,7 +76,7 @@ main figure p{
 
 .input-group{
   display: flex;
-  
+  gap: 7px;
```
Comment (subnitro) [inline style.css:79]: Greate

**Comment by subnitro** [comment]
Greate but i need some time

**Comment by DevSaheerHost** [comment]
Okay ...

**Comment by subnitro** [comment]
Okay this is the time  : )

**Comment by subnitro** [comment]
i accepted

**Comment by DevSaheerHost** [comment]
Thank you 👍

**Pull request merged**